galite-core/src/main/kotlin/org/kopi/galite/util/PlatformFileWriter.kt

```diff
@@ -15,9 +15,93 @@
  * License along with this library; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  */
+
 package org.kopi.galite.util
 
+import java.io.File
+import java.io.FileOutputStream
+import java.io.OutputStream
+
+/**
+ * An class that simplifies file writing
+ */
 class PlatformFileWriter {
 
-}
+  // ----------------------------------------------------------------------
+  // CONSTRUCTOR
+  // ----------------------------------------------------------------------
```

Comment:
Constructors should be like this : 
class PlatformFileWriter(val dataStream: OutputStream,
                         val encoding: String,
                         val lineSeparator: String) {
  /**
   * Sets the number of copy to print
   */
  constructor(file: File,
              encoding: String,
              lineSeparator: String) : this(FileOutputStream(file) , encoding , lineSeparator  ) 
  
  constructor(fileName: String,
              encoding: String,
              lineSeparator: String) : this(File(fileName), encoding, lineSeparator)